### DISCUSSION TITLE
CTB alternative mobile control

### DISCUSSION THREAD

**Discussion by rafroad** [ppy/osu#21029] [Ideas] [answered]
For me the current mobile CTB control is hard to get used to is it possible to replace the control with buttons to move left right and a dash button on top of that so that to dash left the user will press the left move button and the dash button on the right and vice versa
![image](https://user-images.githubusercontent.com/56687086/198987296-982b5760-820e-4aa8-9059-f8464c8def9c.jpeg)
Reference pic

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/discussions/20237#discussioncomment-4015665
